scripts/deploy.sh

```diff
@@ -13,7 +13,17 @@ key="$2"
 public_port=80
 
 scp -o StrictHostKeyChecking=no -i "$key" scripts/remote-deploy.sh "$ssh_hostname":/tmp/remote-deploy.sh
-revision_browser=$(scripts/latest-revision.sh https://github.com/libero/browser.git)
-revision_dummy_api=$(scripts/latest-revision.sh https://github.com/libero/dummy-api.git)
-revision_pattern_library=$(scripts/latest-revision.sh https://github.com/libero/pattern-library.git)
-ssh -o StrictHostKeyChecking=no -i "$key" "$ssh_hostname" PUBLIC_PORT=${public_port} REVISION_BROWSER="$revision_browser" REVISION_DUMMY_API="$revision_dummy_api" REVISION_PATTERN_LIBRARY="$revision_pattern_library" /tmp/remote-deploy.sh
+
+# list of applications
+declare -a applications=(browser dummy-api pattern-library)
+# environment variables string
+declare environment
+for application in "${applications[@]}"
+do
+    revision=$(scripts/latest-revision.sh "https://github.com/libero/${application}.git")
+    # shellcheck disable=SC2018 disable=SC2019
```

Comment:
What do these refer to?

Reply:
The [need for character classes in `tr`](https://github.com/koalaman/shellcheck/wiki/SC2018) for UTF-8 support, despite the fact that applications shouldn't be named piña colada

Reply:
Ah. (I'd be happy with `[:upper:]` `[:lower:]` to avoid the comment: safe, and is just as readable.)

Reply:
Following up in https://github.com/libero/environments/pull/16